aims/aim2.md

```diff
@@ -0,0 +1,23 @@
+# Aim 2
+
+**To evaluate the extent to which low-dimensional representations of single cell data learned by various methods can be used to decompose tissues in the context of autoimmune/rheumatic diseases.**
+
+## Background
+
+Human Cell Atlas' partnership with the Immunological Genome Project (immgenH) will result in single-cell gene expression-based immunocyte phenotyping at unprecedented resolution.
+A compendium comprised of bulk gene expression data from autoimmune/rheumatic diseases is exceptionally well-suited for the biologically-motivated evaluation (e.g., benchmarking) of these immunocyte data.
+The _objective of this aim_ is to build such a compendium and to evaluate methods, including VAEs and other methods developed through this initiative, for defining cell-type-specific expression signatures from single-cell data by measuring their performance in the context of decomposing bulk, whole-tissue autoimmune/rheumatic disease data.
+The quality of the cell-type-specific signatures (and by extension the methods) will be measured by the by using this information in deconvolution frameworks and in correctly identifying cell-type proportion and phenotype in the context of both simulation and real data experiments.
+
+## Approach
+
+We will generate simulated bulk datasets drawn from purified cell lineages, combining the purified cell expression data at different proportions.
+We will use single-cell data-derived cell-type-specific signatures from different methods (e.g., VAEs, other methods developed through this initiative) as input to methods for decomposing bulk data and evaluate performance (e.g., prediction of proportions). 
+Bulk transcriptomes of homogenous cell types perturbed with many stimuli, such as those proposed by Arjun Raj's group, are well-suited for this experiment and will allow us to specifically evaluate performance with respect to cell-type proportion estimates and cell-type phenotype when highly similar cell-type states are added at different proportions.
```

Comment:
I think the answer is _both_.

You can imagine a situation where I have one cell type, let's call it cell type A, in the bulk data under two conditions: unstimulated or treated with a strongly pro-fibrotic stimulus/growth factor/whatever. If I add purified expression data from both `unstimulated cell type A` and `profibrotic cell type A` into a simulated bulk dataset, then I have the same cell type with slightly different phenotypes represented at different proportions. It could be the case that one method for deriving "cell-type-specific" signatures from single-cell data is going to be better at capturing that subtlety than others. That's what I'm trying to get with this eval. I'm making the assumption that there are going to be _similar-enough_ conditions represented in the bulk data and the single-cell data (this entire aim makes that assumption, really). 

Do you think I need a little more here for clarity? Thoughts on how to do that pithily?